api/controllers/CampaignBotController.js

```diff
@@ -366,25 +365,24 @@ class CampaignBotController {
     const dateSubmitted = Date.now();
     const submission = req.signup.draft_reportback_submission;
     submission.submitted_at = dateSubmitted;
+    const currentSignup = req.signup;
```

Comment:
Yes, I'd prefer the first one, but you don't have to do it now, just disable the lint using eslint special comments.
Also, you don't have to add it to each line, you can use block comments as well:

```
/* eslint-disable */
...
/* eslint-enable */
```